src/main/java/net/getsentry/gocd/webhooknotifier/utils/Http.java

```diff
@@ -51,18 +48,21 @@ public static void pingWebhooks(PluginRequest pluginRequest, String type, Object
       return;
     }
 
-    URI[] uris = ps.getWebhookURIs();
-    for (URI uri : uris) {
+    URLAudiencePair[] urlAudiencePairs = ps.getWebhooks();
+    for (URLAudiencePair urlAudiencePair : urlAudiencePairs) {
       try {
         List<Header> headers = new ArrayList<Header>();
-        String url = removeURIUserInfo(uri).toString();
```

Comment:
What was the original purpose of this?

Reply:
To exclude the userInfo from the URI so we could get just the URL (this is a recent change we made in the past few weeks)

Reply:
If it's to exclude userInfo, is there a reason why we needed to remove it?